Userland/Libraries/LibJS/Heap/Handle.h

```diff
@@ -46,7 +46,7 @@ class Handle {
 
     static Handle create(T* cell)
     {
-        return Handle(adopt_ref(*new HandleImpl(cell)));
+        return Handle(adopt_ref(*new HandleImpl(const_cast<RemoveConst<T>*>(cell))));
```

Comment:
There's a handful of `make_handle(const_cast<T>(...))`s already to work around this, can we clean those up in the same commit?

Reply:
Good idea, cleaned!